Makefile

```diff
@@ -225,59 +217,56 @@ intel_SOURCES = \
-  deadcode.mli deadcode.ml \
-  stackframegen.mli stackframegen.ml \
-  stackframe.mli stackframe.ml \
-  linear.mli linear.ml \
-  printlinear.mli printlinear.ml \
-  linearize.mli linearize.ml) \
+  $(addprefix asmcomp/, $(arch_specific_SOURCES)) \
+  asmcomp/arch.mli asmcomp/arch.ml \
+  asmcomp/cmm.mli asmcomp/cmm.ml \
+  asmcomp/printcmm.mli asmcomp/printcmm.ml \
+  asmcomp/reg.mli asmcomp/reg.ml \
+  asmcomp/mach.mli asmcomp/mach.ml \
+  asmcomp/proc.mli asmcomp/proc.ml \
+  asmcomp/strmatch.mli asmcomp/strmatch.ml \
+  asmcomp/cmmgen_state.mli asmcomp/cmmgen_state.ml \
+  asmcomp/cmm_helpers.mli asmcomp/cmm_helpers.ml \
+  asmcomp/afl_instrument.mli asmcomp/afl_instrument.ml \
+  asmcomp/thread_sanitizer.mli asmcomp/thread_sanitizer.ml \
+  asmcomp/cmmgen.mli asmcomp/cmmgen.ml \
+  asmcomp/cmm_invariants.mli asmcomp/cmm_invariants.ml \
+  asmcomp/interval.mli asmcomp/interval.ml \
+  asmcomp/printmach.mli asmcomp/printmach.ml \
+  asmcomp/dataflow.mli asmcomp/dataflow.ml \
+  asmcomp/polling.mli asmcomp/polling.ml \
+  asmcomp/selectgen.mli asmcomp/selectgen.ml \
+  asmcomp/selection.mli asmcomp/selection.ml \
+  asmcomp/comballoc.mli asmcomp/comballoc.ml \
+  asmcomp/CSEgen.mli asmcomp/CSEgen.ml \
+  asmcomp/CSE.mli asmcomp/CSE.ml \
+  asmcomp/liveness.mli asmcomp/liveness.ml \
+  asmcomp/spill.mli asmcomp/spill.ml \
+  asmcomp/split.mli asmcomp/split.ml \
+  asmcomp/interf.mli asmcomp/interf.ml \
+  asmcomp/coloring.mli asmcomp/coloring.ml \
+  asmcomp/linscan.mli asmcomp/linscan.ml \
+  asmcomp/reloadgen.mli asmcomp/reloadgen.ml \
+  asmcomp/reload.mli asmcomp/reload.ml \
+  asmcomp/deadcode.mli asmcomp/deadcode.ml \
+  asmcomp/stackframegen.mli asmcomp/stackframegen.ml \
+  asmcomp/stackframe.mli asmcomp/stackframe.ml \
+  asmcomp/linear.mli asmcomp/linear.ml \
+  asmcomp/printlinear.mli asmcomp/printlinear.ml \
+  asmcomp/linearize.mli asmcomp/linearize.ml \
   file_formats/linear_format.mli file_formats/linear_format.ml \
-$(addprefix asmcomp/, \
-  schedgen.mli schedgen.ml \
-  scheduling.mli scheduling.ml \
-  branch_relaxation.mli branch_relaxation.ml \
-  emitaux.mli emitaux.ml \
-  emit.mli emit.ml \
-  asmgen.mli asmgen.ml \
-  asmlink.mli asmlink.ml \
-  asmlibrarian.mli asmlibrarian.ml \
-  asmpackager.mli asmpackager.ml) \
-$(addprefix driver/, \
-  opterrors.mli opterrors.ml \
-  optcompile.mli optcompile.ml \
-  optmaindriver.mli optmaindriver.ml)
+  asmcomp/schedgen.mli asmcomp/schedgen.ml \
+  asmcomp/scheduling.mli asmcomp/scheduling.ml \
+  asmcomp/branch_relaxation.mli asmcomp/branch_relaxation.ml \
+  asmcomp/emitaux.mli asmcomp/emitaux.ml \
+  asmcomp/emit.mli asmcomp/emit.ml \
+  asmcomp/asmgen.mli asmcomp/asmgen.ml \
+  asmcomp/asmlink.mli asmcomp/asmlink.ml \
+  asmcomp/asmlibrarian.mli asmcomp/asmlibrarian.ml \
+  asmcomp/asmpackager.mli asmcomp/asmpackager.ml \
+  driver/opterrors.mli driver/opterrors.ml \
+  driver/optcompile.mli driver/optcompile.ml \
+  driver/optmaindriver.mli driver/optmaindriver.ml)
```

Comment:
There's a stray parenthesis at the end of the line which I think it's what's breaking the build temporarily